.github/workflows/cleanup-image.yml

```diff
@@ -0,0 +1,13 @@
+name: Remove package versions
```

Comment:
Forresten hva er dette for noe?

Reply:
Den rydder opp i gamle docker images men beholder de siste 5 images (https://github.com/navikt/remove-package-versions). Litt pga avtalen NAV har med GH og hvor mye det koster å beholde alt.

Reply:
Denne funker kun for private repoer. Så denne feila for oss i dette repoet. Må fjerne den herfra